src/components/InputTodo.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FaPlusCircle } from 'react-icons/fa';
+import styles from '../styles/InputTodo.module.css';
+
+const InputTodo = ({ addTodoItem }) => {
+  const [title, setTitle] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleChange = (e) => {
+    setTitle(e.target.value);
+  };
+  InputTodo.propTypes = {
+    addTodoItem: PropTypes.func.isRequired,
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim()) {
+      addTodoItem(title);
+      setTitle('');
+      setMessage('');
+    } else {
+      setMessage('Please add item.');
+    }
+  };
+  return (
+    <>
+      <form onSubmit={handleSubmit} className={styles.formContainer}>
+        <input
+          type="text"
+          placeholder="Add Todo..."
+          value={title}
+          onChange={handleChange}
+          className={styles.inputText}
+        />
+        <button type="button" className={styles.inputSubmit}>
+          {' '}
+          <FaPlusCircle />
+        </button>
```

Comment:
### The submit button of the add todo form is not working
I would suggest you change the value of the button type attribute to `submit`.

![image](https://user-images.githubusercontent.com/43586022/220968675-d1cf380b-9bd2-4572-af66-b2b225f2f949.png)